sys/net/gnrc/network_layer/ipv6/nib/_nib-slaac.c

```diff
@@ -96,7 +138,91 @@ void _auto_configure_addr(gnrc_netif_t *netif, const ipv6_addr_t *pfx,
 }
 #endif  /* CONFIG_GNRC_IPV6_NIB_6LN || CONFIG_GNRC_IPV6_NIB_SLAAC */
 
+#if IS_ACTIVE(CONFIG_GNRC_IPV6_STABLE_PRIVACY)
+bool _iid_is_iana_reserved(const eui64_t *iid)
+{
+    return (iid->uint64.u64 == htonll(0))
+           || (iid->uint32[0].u32 == htonl(0x02005eff) && iid->uint8[4] == 0xfe)
+           || (iid->uint32[0].u32 == htonl(0xfdffffff) && iid->uint16[2].u16 == htons(0xffff) && iid->uint8[6] == 0xff && (iid->uint8[7] & 0x80));
+}
+
+inline bool _stable_privacy_should_retry_idgen(uint8_t *dad_ctr, const char *reason) {
+    if (*dad_ctr < STABLE_PRIVACY_IDGEN_RETRIES) { //within retry limit
```

Comment:
At the very last stages of review, `//` style comments will be flagged for code style. Sorry it's not automated or you'd have seen it in CI :-/